### ISSUE TITLE
Crash in 64 bits version

### ISSUE THREAD

**Issue by jplu** [puku0x/cvdrone#39]
Hello,

An issue is coming from a switch from 32 bits to 64 bits.
Here what I did to make you able to reproduce the issue:

* Compile OpenCV 3.1 in 64 bits
* Compile FFMPEG in 64 bits
* Unzip the 64 bits library for pthread from the provided zip
* Open the Visual Studio solution
* Switch in the settings to 64 bits compilation
* Change the dependencie's name to link
* Compile and then execute

I did these steps with both Visual Studio 2012 and 2015 to get the below error:

```
The application was unable to start correctly (0x000007b).
Click on OK to close the application.
```

Regards.

**Comment by puku0x** [comment]
Thank you very much !
I will prepare for it.

**Comment by puku0x** [comment]
Hmm,
http://answers.microsoft.com/en-us/windows/forum/windows_7-performance/error-the-application-was-unable-to-start/05a2b904-3f61-4d08-94d6-e2ff92161111?auth=1

**Comment by jplu** [comment]
Yep, I came up to that thread as well, but I checked what they say and all my DLLs for OpenCV, FFMPEG and Pthread are in 64 bits :(

**Comment by JAbdor7** [comment]
Good day,

I was wondering if you were able to compile the cvdrone library for x64 in visual studio?

**Comment by jplu** [comment]
No I gave up :(

**Comment by JAbdor7** [comment]
I think I found the solution. There seems to be an issue with the Linker Manifest File. After I turn that off I don't get the error. Now since I don't have the drone with me I can't test it, however I do get an error that it can't connect to the drone. I'll post if I am able to get it working in two days once i have access to the drone.

**Comment by Youssef-Salama** [comment]
@JAbdor7 I have the drone with me. If you could forward me the code along with instructions on how to run it, I will test it and let you know. I need this urgently for a college research. Thank you.

**Comment by JAbdor7** [comment]
I was able to run cvdrone on x64 and it was tested with the drone.
